python/python/glide/async_commands/transaction.py

```diff
@@ -4611,6 +4611,7 @@ def sort(
         """
         Sorts the elements in the list, set, or sorted set at `key` and returns the result.
         The `sort` command can be used to sort elements based on different criteria and apply transformations on sorted elements.
+        This command is routed to primary nodes only.
```

Comment:
I guess you don't need to add it here - the command isn't routed there, an entire transaction is routed ... according to the first command it has.
If you want to have such notice in transaction implementation, please reword it to reflect the actual behavior.